### ISSUE TITLE
screen reader mode rarely reads command output with VoiceOver

### ISSUE THREAD

**Issue by dmartin** [xtermjs/xterm.js#3247]
When `screenReaderMode` is enabled, the output of an entered command is rarely read back by VoiceOver (e.g. is added to the live region in the DOM.)

The length of the command's output seems to determine how likely it is that the output will actually be read, for example `pwd` output will almost never be read, `ls` output maybe one in twenty times, `ls -l` output most of the time. This seems like it might be a race condition that depends on how long it takes to render the output of a command.

For some reason, I have never had this problem with VSCode's integrated terminal, only when using xterm.js standalone.

## Details
- Browser and browser version: Chrome 89, Safari 14.0.3
- OS version: macOS 11.2
- xterm.js version: 4.10

### Steps to reproduce

1. Clone and set up xterm.js
2. In demo page, enable `screenReaderMode` and VoiceOver
3. Note that entered text is spoken, but the results of commands only rarely.

**Comment by dmartin** [comment]
Further pointing to what looks like a race condition, prefixing commands with `sleep 0.1;` ensures that their output is always read.

**Comment by casey-speer** [comment]
I'm seeing the same behavior with `4.6.0` (our current version) and `4.16.0` (latest version). Any updates on a fix? Curious because we'd really love to have solid screen reader support in our terminal application.

We may be able to help contribute toward a fix if one is not on the horizon.

**Comment by Tyriar** [comment]
I was mainly targeting NVDA when I worked on the screen reader support, I'm guessing VoiceOver probably will work but you might need to tweak some of the verbosity settings? Any help in this area would be appreciated.

**Comment by dmartin** [comment]
On current master / macOS 12.2, reverting* [this commit](https://github.com/xtermjs/xterm.js/commit/5dc57a1305cd7eca10bf01793645441f711c2048) appears to fix the issue.

There is a "stutter" where the `tooMuchOutput` string is interrupted and read twice if present (might be a separate bug), but all output does at least get read.

\* Note that it doesn't cleanly `git revert` since the file was moved into the `browser` subdirectory.

**Comment by casey-speer** [comment]
@Tyriar Thanks for getting back back to me. I've been using VoiceOver with high verbosity and have still been seeing this issue, but still working on a good way to directly test our setup on NVDA. Good to know this seems restricted to VO.

@dmartin thanks for the context. We'll help fix if possible, and forking with that revert might help us in the interim.

Appreciate the quick responses!

**Comment by Tyriar** [comment]
@dmartin interesting, it was an awfully long time ago but that was meant to improve things specifically on macOS 🤔

**Comment by ziwangj** [comment]
Have same issue on xterm.js version: 5.1.0. Revert [this commit](https://github.com/xtermjs/xterm.js/commit/5dc57a1305cd7eca10bf01793645441f711c2048) is not work for me. If append sleep 1 on each command, the output can be read properly, but we cannot use that in reality. Any turnaround fix for that?
